app/views/api/v2/hosts/main.json.rabl

```diff
@@ -9,7 +9,10 @@ attributes :ip, :environment_id, :environment_name, :last_report, :mac, :realm_i
            :enabled, :puppet_ca_proxy_id, :managed, :use_image, :image_file, :uuid, :compute_resource_id, :compute_resource_name,
            :compute_profile_id, :compute_profile_name, :capabilities, :provision_method,
            :puppet_proxy_id, :certname, :image_id, :image_name, :created_at, :updated_at,
-           :last_compile, :puppet_status
+           :last_compile, :global_status, :global_status_label, :puppet_status
+
+attributes :configuration_status, :configuration_status_label, :if => @host && @host.get_status(HostStatus::ConfigurationStatus).relevant?
```

Comment:
Commented on this last time, but I think these should move into the show rabl if they're not meant to be displayed on the index.